site/en/blog/private-network-access-update/index.md

```diff
@@ -6,10 +6,17 @@ authors:
   - titouan
 description: Chrome has some updates to share about the standardization of the Private Network Access - formerly known as CORS-RFC1918 - plans to restrict private network resource accesses, and how your websites can align with those changes.
 date: 2021-03-11
+updated: 2021-06-23
 hero: image/YLflGBAPWecgtKJLqCJHSzHqe2J2/dwtN0NkxkBmIz1EyhzAm.jpg
 alt: A private sign in German
 ---
 
+**Updates**
+
+- **June 23, 2021**: Based on feedback reported we've decided to adjust the
+  timeline for private network requests initiated from non-secure contexts will
+  be blocked in Chrome 93 instead of 92.
```

Comment:
```suggestion
- **June 23, 2021**: Based on the feedback, the timeline for
  blocking private network requests initiated from non-secure contexts has been changed.
  Such requests will be blocked in Chrome 93 instead of 92.
```